src/content/en/fundamentals/media/manipulating/cheatsheet.md

```diff
@@ -0,0 +1,372 @@
+project_path: /web/_project.yaml
+book_path: /web/fundamentals/_book.yaml
+description: A summary of commands used to convert a raw mov file to an encrypted full HD file for web playback.
+
+{# wf_updated_on: 2017-06-05 #}
+{# wf_published_on: 2017-06-05 #}
+
+# Media Manipulation Cheat Sheet  {: .page-title }
+
+{% include "web/_shared/contributors/josephmedley.html" %}
+
+This document shows in order commands needed to get from a raw mov file to
+encrypted assets packaged for DASH or HLS. For the sake of having a goal to
+illustrate, I'm converting my source file to a bitrate of 8Mbs at a resolution
+of 1080p (1920 x 1980). Adjust these values as your needs dictate.
+ 
+Conversion is done with two applications:
+[Shaka Packager](https://github.com/google/shaka-packager) and
+[ffmpeg](https://ffmpeg.org/download.html). Although
+I've tried to show equivalent operations for all procedures, not all operations
+are possible in both applications.
+ 
+In many cases, commands may be combined in a single command line operation, and
+in actual use would be. For example, there's nothing preventing you from setting
+an output file's bit rate in the same operation as a file conversion. For this
```

Comment:
nit: bit rate vs bitrate. I don't have a particular feel on which is better, but I guess we need to pick one and stick.